code/src/components/RemoveCompleted.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { tasks } from 'reducers/tasks';
+
+import { Button, Emoji } from 'styles/GlobalStyles';
+
+export const RemoveCompleted = () => {
+  const items = useSelector((store) => store.tasks.items);
+  const enabled = items.length > 0;
```

Comment:
Shouldn't this be enabled only if there are complete items?

Reply:
Of course! Thanks :D